code/game/objects/items/weapons/weaponry.dm

```diff
@@ -475,8 +475,9 @@
 	icon_state = "baseball_bat"
 	item_state = "baseball_bat0"
 	inhand_states = list("left_hand" = 'icons/mob/in-hand/left/swords_axes.dmi', "right_hand" = 'icons/mob/in-hand/right/swords_axes.dmi')
+	autoignition_temperature = AUTOIGNITION_WOOD
 	flags = TWOHANDABLE
-	force = 15
+	force = 14
```

Comment:
if you want to nerf its damage, consider that it is as bulky as a toolbox
since it's a wielded weapon, why not give it a spear treatment where it has higher damage difference?

Reply:
Should I make it random between a range of 13 and 16?

Reply:
that's an option, but i meant that it could have less damage when unwielded, but the same amount for wielded